### DISCUSSION TITLE
Why isn't it on the Play Store?

### DISCUSSION THREAD

**Discussion by ravenfeld** [Helium314/SCEE#563] [General]
I discovered the application by word of mouth during a state of the map in France. 
I have the impression that it could have more users if it were also distributed on the Play Store.

Maybe you'd prefer us to talk about it on the various forums to get the word out?

**Comment by Helium314** [comment]
I don't want to waste my time on messaging with Google bots, see e.g. https://github.com/streetcomplete/StreetComplete/issues/3466 or https://github.com/streetcomplete/StreetComplete/issues/3768 and possibly a bunch more issues.
Also I don't have a Google account.

So far I avoided trying to promote this app because it's much more dangerous than SC in the hands of users new to OSM.

**Reply by ravenfeld** [reply]
I didn't understand the robot stuff,
But I do. Since you don't have a Google account.
For the promotion, I'm thinking of putting it in the same section as JOSM for the time being. 
I'll make a post about it on the French group because I think it's what a lot of people are looking for, at least in my circle of friends.

**Reply by HolgerJeromin** [reply]
> putting it in the same section as JOSM 

Not in my opinion.
SCEE feels like easy to understandable, but needs knowledge.
JOSM is clear to have a learning curve

**Reply by ravenfeld** [reply]
I'll try to make sure I mention it in the post on the French forum.

**Reply by mnalis** [reply]
I don't know the intended format, but I'd at the very beginning try to explain that "SCEE" stands for  _"StreetComplete **Expert** Edition"_, and if someone **considers themselves an OSM expert** (i.e. they know raw tag values by heart, can troubleshoot OSM issues themselves, etc.), **they** might enjoy it quite much as it provides much extra functionality and shortcuts for doing thing faster with less work. Otherwise, keep scrolling!
  
But while from outside SCEE might look as easy as StreetComplete, those extra powers come with a caveat -- it allows user to easily **unintentionally damage the OSM map** if they've not been truthful to themselves about their "expert" status (see https://github.com/Helium314/SCEE/issues/365#issuecomment-1369144337 for more details). That NASCAR vehicle might look somewhat similar to the regular family car from afar, but while it can go to wanted destination much faster in hands of an expert driver, it also requires much more skill and experience to use safely.

And JOSM is more, like, Space Shuttle. There is no chance you could ever mistakenly get the idea you can safely drive it to the intended destination without first spending a decade doing training sims first :smiley:
